src/EmitJs.re

```diff
@@ -1343,5 +1349,7 @@ let emitTranslationAsString =
        ~config,
        ~requires=finalEnv.requires,
      )
-  |> Emitters.toString(~separator="\n\n");
-};
\ No newline at end of file
+  |> Emitters.toStrings
+  |> List.append(_, moduleItems)
```

Comment:
How about keeping the same style as the previous cases:
```reason
let emitters = moduleItemsEmitter |> emitAllModuleItems(~emitters)
```
which takes care of whatever it needs to do with lists etc internally, and uses e.g. `Emitters.export`.

Reply:
this was an super nice suggestion, could delete most of my custom code